package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Products and services relating to wellcomecollection.org.",
   "scripts": {
+    "setupCommon": "yarn install --production && yarn flow && cd common && yarn && yarn install",
```

Comment:
Do we need to run the install in common too? I thought the install at the root level would be fine?

Reply:
I'm not certain. I think that circle would previously have run both installs looking at:

https://github.com/wellcomecollection/wellcomecollection.org/blob/7f97f6c03a2d455d32d0e6b95ecadcc0f3413cc4/.circleci/config.yml#L140 

`common_tests` does a yarn install in common, and `install_modules` does the yarn install in the root.

I will run an experiment to see if it's really required.

Reply:
Turns out _no_, we do not need to do this. 👍

Reply:
Probably added when doing the work around including `common` and added this as a "I wonder if this will make it work" moment.